tools/csv-merger/csv-merger.go

```diff
@@ -749,6 +750,12 @@ func injectCSVRelatedImages(r *unstructured.Unstructured) error {
 			"image": *noobaaDBContainerImage,
 		})
 	}
+	if *mustGatherImage != "" {
+		relatedImages = append(relatedImages, map[string]interface{}{
+			"name":  "must-gather",
```

Comment:
let's call this `ocs-must-gather` just to be clear and not confuse with `ocp-must-gather`